AspNetCore.Diagnostics.HealthChecks.sln

```diff
@@ -1,978 +1,994 @@
-﻿
```

Comment:
Probably you have changed file encoding so the diff is unreadable.

Reply:
Fixed, I accidentally set it to `CRLF`